less/components/footer.less

```diff
@@ -0,0 +1,92 @@
+footer#solstice-footer {
+  .menu-heading {
+    .text-semibold;
+    margin-bottom: 1rem;
+
+    &::after,
+    &::before {
+      display: none;
+    }
+  }
+
+  section {
+    z-index: 0;
+  }
+
+  .footer-legal-wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 1rem;
+
+    @media (max-width: @screen-sm-max) {
+      clear: both;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+    }
+
+    &::before,
+    &::after {
+      display: none;
+    }
+
+    .logo {
+      max-width: 180px;
+    }
+  }
+
+  section#footer-legal {
+    .menu-heading {
+      display: none;
+    }
+
+    ul {
+      .list-inline;
+      margin-bottom: 0;
+
+      a {
+        padding: 0;
+      }
+    }
+  }
+
+  .footer-end-newsletter {
+    padding: 3rem;
+    border: 1px solid @brand-primary-orange;
+    border-radius: 20px;
+
+    @media (max-width: @screen-sm-max) {
```

Comment:
Is there a reason we are using desktop-first CSS here? We tend to use mobile-first wherever possible.